lib/hyperion/aux/logger.rb

```diff
@@ -23,7 +27,16 @@ def log_stub(rule)
     private
 
     def log_headers(headers, logger)
-      headers.each_pair { |k, v| logger.info "    #{k}: #{v}" unless k == 'Expect' }
+      h = headers.keep_if { |k| k != 'Expect' }
+      logger.info(log_message(h))
+    end
+
+    def log_message(obj)
```

Comment:
that's only true if the response is a 4xx. see https://github.com/indigobio/hyperion/blob/f-logs/lib/hyperion/result_handling/result_maker.rb#L53-L56